libpod/container_log.go

```diff
@@ -4,10 +4,12 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"time"
 
 	"github.com/containers/podman/v3/libpod/define"
 	"github.com/containers/podman/v3/libpod/events"
 	"github.com/containers/podman/v3/libpod/logs"
+	"github.com/nxadm/tail/watch"
```

Comment:
same here

Reply:
Good catch!  Looks like my vim plugin chose the wrong dep.  Thanks!